_docs/developer/how_to_contribute.md

```diff
@@ -28,12 +28,26 @@ Be sure to read the [Suggestions for New Developers](/developer#suggestions-for-
 
 * The PR title should be useful and descriptive (not just the issue#).
 
-* Titles of PR, Issues, and commits should be <= 50 characters.
-  *Usually*.
+    * Titles of PR, Issues, and commits should be <= 50 characters.  *Usually*.
+
+    * Suggested format of the PR Title:
+
+      ```
+      [<TYPE>:<MODULE>] <SUBJECT> (#<PR NUMBER>)
```

Comment:
We should probably remove the `(#<PR Number>)` as it's not something we need people to manually append to the title themselves, and that it should be automatically appended when going to squash + merge. Might want to make a note to make sure it's included there though.

Reply:
Yes, See note a few lines down :)

Reply:
`(#<PR NUMBER>)` is appended when someone does a squash + merge. If we are not going to put `(#<PR NUMBER>)` in PR titles (and I don't think we should), I would suggest mentioning this in other places and specifying that it is for squash commit message.

However it may be just okay if we say nothing about `#<PR NUMBER>` at all because all pull requests are merged by you (Barb and Matt). :)